mono/mini/mini-wasm-debugger.c

```diff
@@ -1281,6 +1284,19 @@ describe_variables_on_frame (MonoStackFrameInfo *info, MonoContext *ctx, gpointe
 	return TRUE;
 }
 
+EMSCRIPTEN_KEEPALIVE void
+mono_wasm_get_deref_ptr_value (void *value_addr, MonoClass *klass)
+{
+	MonoType *type = m_class_get_byval_arg (klass);
+	if (type->type != MONO_TYPE_PTR && type->type != MONO_TYPE_FNPTR) {
+		DEBUG_PRINTF (2, "BUG: mono_wasm_get_deref_ptr_value: Expected to get a ptr type, but got 0x%x\n", type->type);
+		return;
```

Comment:
is the rest of the app going to be well-behaved if one of the get_ methods just silently fails to describe the value as expected? the other ones don't do this

Reply:
Hm we need to appropriately report errors in similar cases. Currently, we just return empty results. I will fix this for such calls, in a future PR. For this, I'm adding a test just to check that this doesn't crash.